nengo/tests/conftest.py

```diff
@@ -111,10 +137,13 @@ def pytest_generate_tests(metafunc):
 
 
 def pytest_addoption(parser):
-    parser.addoption('--benchmarks', action='store_true', default=False,
-                     help='Also run benchmarking tests')
-    parser.addoption('--plots', action='store_true', default=False,
-                     help='Also run plotting tests')
+    parser.addoption(
+        '--plots', nargs='?', default=False, const=True,
+        help='Save plots (optional with directory to save them in).')
+    parser.addoption(
+        '--benchmarks', nargs='?', default=False, const=True,
```

Comment:
I would call this option `--analytics`, since it's not specific to benchmark tests (i.e., it controls whether the analytics fixture does anything in all tests). Also replace "benchmarks" with "analytics" pretty much everywhere else (e.g. "nengo.simulator.benchmarks" becomes "nengo.simulator.analytics").